lib/engine/game/g_18_mex/step/buy_sell_par_shares.rb

```diff
@@ -40,13 +40,14 @@ def attempt_ndm_action_on_unavailable?(bundle)
             bundle.corporation == @game.ndm && @game.phase.status.include?('ndm_unavailable')
           end
 
-          def room_to_gain?(entity, bundle)
+          def room_to_gain?(entity, bundle, corporation)
```

Comment:
Since `corporation` is always equal to `bundle.corporation` I wouldn't add a new argument that adds no additional information to the function call (having it makes it seem like it's possible for `corporation` to not be deducible from `bundle`). It's not like dereferencing `bundle.corporation` another time or two will have a performance impact.

Reply:
@dfannius Have now refactored and removed that parameter.